paddle/framework/op_registry.h

```diff
@@ -274,6 +285,24 @@ class OpRegistry {
     return CreateOp(op_desc.type(), inputs, outputs, attrs);
   }
 
+  static OperatorPtr CreateGradOp(OperatorPtr op) {
```

Comment:
we named every Grad operator in XXXGrad, so we create interface named in `CreateGradOp`